src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -194,6 +195,12 @@ public function load(array $configs, ContainerBuilder $container)
             }
         }
 
+        // If the slugger is used but the String component is not available, we should throw an error
+        if (!class_exists(SluggerInterface::class)) {
+            $container->register('slugger', 'stdClass')
```

Comment:
stdClass ?

Reply:
What's your question?

Reply:
a class is required, but any class works, so stdClass or whatever :)